lib/utils.js

```diff
@@ -107,6 +107,21 @@ function appendStubVarsToDotenv (generator, label, vars) {
   generator.fs.append(file, content)
 }
 
+function appendVarsToDotenv (generator, label, envVariable, value) {
+  const searchString = envVariable
+  const re = new RegExp('^.*' + searchString + '.*$', 'gm')
+  const file = generator.destinationPath(dotenvFilename)
+  const prevContent = (generator.fs.exists(file) || '') && generator.fs.read(file)
```

Comment:
for maintainability, this line should be simplified (and perhaps expanded instead of compressed here on one line). The rules of the logical and operator in this may or may not be readily undestood by maintainers

```

```